bindings/jaxrs/src/main/java/org/commonjava/indy/core/bind/jaxrs/ContentAccessHandler.java

```diff
@@ -174,7 +174,7 @@ public Response doHead( String type, String name, String path, Boolean cacheOnly
         {
             try
             {
-                Transfer item;
+                Transfer item = null;
```

Comment:
will this lead to a 404 response on line 204 if we don't assign it somehow (or change the condition on line 191) in the case of an existence check?

Or, maybe a better way to put is: Why doesn't this generate a 404 in line 204?